tests/e2e/cypress/integration/Autologin/common.ts

```diff
@@ -0,0 +1,55 @@
+import { setUserTokenApiV1 } from '../../support/centreonData';
+import {
+  applyConfigurationViaClapi,
+  executeActionViaClapi,
+  insertFixture,
+} from '../../commons';
+
+const initializeContactData = (): Cypress.Chainable => {
+  const files = ['resources/clapi/contact1/01-add.json'];
+
+  return cy.wrap(Promise.all(files.map(insertFixture)));
+};
+
+const insertContactFixture = (): Cypress.Chainable => {
+  return initializeContactData()
+    .then(applyConfigurationViaClapi)
+    .then(() => cy.visit(`${Cypress.config().baseUrl}`))
+    .then(() => cy.fixture('users/admin.json'));
+};
+
+const getIframeDocument = () => {
+  return cy.get('iframe').its('0.contentDocument').should('exist');
+};
+
+const getIframeBody = () => {
+  return getIframeDocument()
+    .its('body')
+    .should('not.be.undefined')
+    .then(cy.wrap);
+};
```

Comment:
It's better to define it as a cypress custom command: https://github.com/centreon/centreon-autodiscovery/blob/develop/tests/e2e/cypress/support/commands.ts#L1-L7